core/tee/tee_svc_storage.c

```diff
@@ -299,9 +307,8 @@ TEE_Result syscall_storage_obj_open(unsigned long storage_id, void *object_id,
 		goto err;
 	}
 
-	o->info.handleFlags =
-	    TEE_HANDLE_FLAG_PERSISTENT | TEE_HANDLE_FLAG_INITIALIZED;
-	o->flags = flags;
+	o->info.handleFlags = TEE_HANDLE_FLAG_PERSISTENT |
+			      TEE_HANDLE_FLAG_INITIALIZED | flags;
```

Comment:
Now that we store the `TEE_DATA_*` flags along with the `TEE_HANDLE_*` flags, the name `handleFlags` seems badly chosen. How about renaming it to just `flags` with a comment `/* TEE_HANDLE_* and TEE_DATA_* */` perhaps?

Reply:
`handleFlags` is a field in `TEE_ObjectInfo`, which is defined by the GP API so we we're more or less stuck with that.

Reply:
Duh! Yes of course. We would never have used camelCase in the core OP-TEE code ;-)